### PR TITLE
Improve Vapor pressure, and Enthalpy of Vaporization behavior of SPSDK 

### PR DESCRIPTION
**Pull request by n7275**

### CODE REVIEW

File: Orbitersdk/samples/ProjectApollo/src_sys/PanelSDK/Internals/Hsystems.cpp [hunk 1]
```diff
@@ -326,31 +326,50 @@ void h_substance::operator -=(h_substance add) {
 	}
 }
 
+double h_substance::VAPENTH()
+{
+	if(Temp > CRITICAL_T[subst_type] || Temp <= 0.0){return 0.0;}
```
Comment (ThymoNL) [inline Orbitersdk/samples/ProjectApollo/src_sys/PanelSDK/Internals/Hsystems.cpp]: Curly brackets aren't strictly necessary here.

**Review by ThymoNL — DISMISSED** [review]

File: Orbitersdk/samples/ProjectApollo/src_sys/PanelSDK/Internals/Hsystems.cpp [hunk 2]
```diff
@@ -326,31 +326,50 @@ void h_substance::operator -=(h_substance add) {
 	}
 }
 
+double h_substance::VAPENTH()
```
Comment (ThymoNL) [inline Orbitersdk/samples/ProjectApollo/src_sys/PanelSDK/Internals/Hsystems.cpp]: I don't see any assignments. This function could be made constant

**Review by ThymoNL — DISMISSED** [review]

**Review by indy91 — APPROVED** [review]

**Comment by rcflyinghokie** [comment]
I have tested these changes in 3 complete missions, they seem to work properly with no ill effects.

**Review by rcflyinghokie — APPROVED** [review]
I have tested these changes in 3 complete missions, they seem to work properly with no ill effects with the current state of NASSP.

**Pull request merged**